app/src/main/java/org/gdg/frisbee/android/onboarding/FirstStartStep1Fragment.java

```diff
@@ -53,21 +60,39 @@
 
     private static final String ARG_SELECTED_CHAPTER = "selected_chapter";
     @Bind(R.id.chapter_spinner)
-    Spinner mChapterSpinner;
+    AutoCompleteSpinnerView autoCompleteSpinnerView;
     @Bind(R.id.confirm)
-    Button mConfirm;
+    Button mConfirmButton;
     @Bind(R.id.viewSwitcher)
     ViewSwitcher mLoadSwitcher;
-    private ChapterAdapter mSpinnerAdapter;
+    private ChapterAdapter mChapterForCityName;
```

Comment:
Why do we need 2 adapters? 
Since we have auto complete now, I think location button can just find the nearest chapter and write its name to the `EditText`. If users want to change it, they can just delete it.